cmd/manager/main.go

```diff
@@ -88,7 +88,7 @@ func main() {
 		os.Exit(1)
 	}
 
-	resyncPeriod := time.Duration(syncPeriodSeconds) * time.Second
+	resyncPeriod := time.Duration(*syncPeriodSeconds) * time.Second
```

Comment:
I don’t think the controller-runtime manager `SyncPeriod` should be set the same as picchu’s requeue interval. From [the controller-runtime docs](https://godoc.org/sigs.k8s.io/controller-runtime/pkg/manager#Options):

> SyncPeriod determines the minimum frequency at which watched resources are reconciled. A lower period will correct entropy more quickly, but reduce responsiveness to change if there are many watched resources. Change this value only if you know what you are doing. *Defaults to 10 hours* if unset.

idk why operator-sdk generates a default resync period so wildly shorter from the default recommendation, but it seems like we should set `SyncPeriod` much higher than the interval we want to re-reconcile our resources.